src/dsp_tools/xmllib/models/dsp_base_resources.py

```diff
@@ -51,7 +51,7 @@ def create_new(
     ) -> AnnotationResource:
         """
         Creates a new annotation resource.
-        An annotation provides metadata to another resource.
+        An annotation is a comment to another resource.
```

Comment:
Noémi and me, we improved the docstrings of the DSP base resources, to improve understandability